content/admin/monitoring-managing-and-updating-your-instance/updating-the-virtual-machine-and-physical-resources/increasing-cpu-or-memory-resources.md

```diff
@@ -123,3 +123,15 @@ To increase the resources available to a {% data variables.product.prodname_ghe_
 {% data reusables.enterprise_installation.configuration-recognized %}
 {% data reusables.enterprise_installation.increase-resources-test-in-maintenance-mode %}
 {% data reusables.enterprise_installation.increase-resources-disable-message-and-maintenance-mode %}
+
+## Adding CPU or memory resources for Google Cloud Platform
+
+### Resizing considerations for Google Cloud Platform
+
+To add CPU or memory resources for an instance on Google Cloud, you must change the instance's size, have access to your company's Google Cloud infrastructure, and be familiar with  Google Cloud Run to manage Google Cloud instances.
```

Comment:
- I'd recommend moving this up, under the first new heading you've added.
- We want this to be a high-level overview of what the customer needs and must do. Offhand, I'll say that offhand, I don't _think_ Google Cloud Run is involved in the launch of an instance for GHES in GCP 🤔

Reply:
Thanks @mattpollard! :) I picked GCP because in my research, this is what is commonly used.

Reply:
I could use some help here @nguyenalex836